lib/matplotlib/lines.pyi

```diff
@@ -130,8 +130,8 @@ class AxLine(Line2D):
     def get_xy1(self) -> tuple[float, float] | None: ...
     def get_xy2(self) -> tuple[float, float] | None: ...
     def get_slope(self) -> float: ...
-    def set_xy1(self, x: float, y: float) -> None: ...
-    def set_xy2(self, x: float, y: float) -> None: ...
+    def set_xy1(self, xy1: tuple[float, float]) -> None: ...  # type: ignore[override]
+    def set_xy2(self, xy2: tuple[float, float]) -> None: ...  # type: ignore[override]
```

Comment:
Why are these ignore comments here?  mypy docs discuss using this if the [signature is different from that of the method on the parent class](https://mypy.readthedocs.io/en/stable/common_issues.html#incompatible-overrides), but `set_xy1` and `set_xy2` do not appear to exist on the parents.

Reply:
I... don't know anything about type-checked python (the less said about it the better because it'd devolve into ranting *extremely* quickly) and just cargo-culted these two lines.  TBH I don't even know if they are necessary, I just suspect they are...

Reply:
The type ignore should not be necessary here, especially with the override specifier.

Reply:
So how should that exactly be spelled?  Can I just remove the comment?

Reply:
Just drop the comments.

Reply:
Removed.